### ISSUE TITLE
Parametrize functions in wikimedia-requests/requests.js

### ISSUE THREAD

**Issue by kpagacz** [kpagacz/wiki-on-fire#128]
Currently there are two get requests in the file, which print to the console.

Please, refactor them into two functions, parametrized by date.

The acceptance critera includes creating one or more classes which represent the single item returned from the endpoints. The idea is to map the response from the server into something which can be later manipulated without knowing the underlying structure of the data submitted in the response.

In short:
* refactor functions
* create classes to represent the response data
